.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
These are local files and should be ignored by Git, kindly add them to the `.gitignore` file.